index.html

```diff
@@ -44,5 +44,178 @@ <h2><img src="image/LET’S CONNECT.svg" alt=""></h2>
+                    sign-ups required.
+                </p>
+                <ul class="tech skills">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+
+            <div class="my-work">
+                <img src="image/portfolio6.svg" alt="image" />
+                <h2>Multi-Post Stories</h2>
+                <div class="position-container">
+                    <span>Canopy</span>
+                    <ul>
+                        <li class="developed marked">Back End Dev</li>
+                        <li class="year marked">2022</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="tech skills">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+
+            <div class="my-work">
+                <img src="image/Portfolio7.svg" alt="image" />
+                <h2>Tonic</h2>
+                <div class="position-container">
+                    <span>Canopy</span>
+                    <ul>
+                        <li class="developed marked">Back End Dev</li>
+                        <li class="year marked">2022</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="tech">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+
+            <div class="my-work">
+                <img src="image/Portfolio8.svg" alt="image4" />
+                <h2>Multi-Post Stories</h2>
+                <div class="position-container">
+                    <span>Canopy</span>
+                    <ul>
+                        <li class="developed marked">Back End Dev</li>
+                        <li class="year marked">2022</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="tech skills">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+        </section>
+        <section class="section3">
+            <div class="about">
+                <h1>About Myself</h1>
```

Comment:
- [ ] This is the title of this section, but not the title of the page. Usually, the page has one main title and no more than one `h1` tag. Please change this to a `h2` and read [this article](https://www.reliablesoft.net/h1-tag/) for a better understanding of the headings and SEO.